templates/cicd/pipeline_cfn.yml

```diff
@@ -30,6 +30,9 @@ Resources:
             Action:
               - sts:AssumeRole
       Path: /
+      ManagedPolicyArns:
+        - 'arn:aws:iam::aws:policy/AmazonSSMReadOnlyAccess'
+        - 'arn:aws:iam::aws:policy/AWSCloudFormationReadOnlyAccess'
```

Comment:
nit: Should we add comments explaining that SSM is for "archer env ls" and cfn read is for "archer app package"?